daemon/src/engine/sys/con_curses.cpp

```diff
@@ -66,7 +66,9 @@ static bool     forceRedraw = false;
 static int      stderr_fd;
 #endif
 
-static const int LOG_LINES = ( LINES - 3 );
+static int NumLogLines() {
```

Comment:
I'm sure it will be inlined anyway. Everyone says the inline keyword is an anachronism.